src/latex/types.ml

```diff
@@ -23,14 +23,19 @@ type elt =
   | List of list_info
   | Description of (t * t) list
   | Indented of t
+  | Layout_table of layout_table
   | Table of table
   | Ligaturable of string
 
 and section = { level : int; label : string option; content : t }
 
 and list_info = { typ : Odoc_document.Types.Block.list_type; items : t list }
 
-and table = { row_size : row_size; tbl : t list list }
+and layout_table = { row_size : row_size; tbl : t list list }
+
+and alignment = Default | Left | Right | Center
```

Comment:
Why not using the same option type as in `document/types.ml` to stay consistent with the html generator?